haystack/pipelines/base.py

```diff
@@ -379,6 +381,8 @@ def eval(
                         If you want to pass a param to all nodes, you can just use: {"top_k":10}
                         If you want to pass it to targeted nodes, you can do:
                         {"Retriever": {"top_k": 10}, "Reader": {"top_k": 3, "debug": True}}
+            :param sas_model_name_or_path: SentenceTransformers semantic textual similarity model to be used for sas value calculation, 
```

Comment:
Not sure what's the best place but somewhere we need to document the columns of the dataframe that we are building - either here or in `EvalResults` (e.g. What does `answer_match` in the retriever df mean?)

Reply:
done in `EvaluationResult`